crates/common/tedge_config/src/tedge_toml/tedge_config.rs

```diff
@@ -150,10 +156,15 @@ define_tedge_config! {
             #[tedge_config(example = "123456", default(value = "123456"))]
             pin: Arc<str>,
 
-            /// A URI of the token/object to use.
+            /// A URI of the token/object to use by tedge-p11-server.
```

Comment:
Just minor readability improvement.

```suggestion
            /// A URI of the token/object to be used by tedge-p11-server.
```

Reply:
fixed 1166b7420c